engine/chain/internal/bhs/bhs_mock_test.go

```diff
@@ -0,0 +1,52 @@
+package bhs_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/bitcoin-sv/go-paymail/spv"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/go-resty/resty/v2"
+	"github.com/jarcoal/httpmock"
+)
+
+const (
+	bhsURL   = "http://localhost:8090"
```

Comment:
I think port should be 8080, but it's a mock so not sure if really an issue

Reply:
On my local machine I have `8090`.
Will change to default one 👍